src/aks-preview/azext_aks_preview/_params.py

```diff
@@ -84,6 +84,7 @@ def load_arguments(self, _):
         c.argument('workspace_resource_id')
         c.argument('skip_subnet_role_assignment', action='store_true')
         c.argument('enable_cluster_autoscaler', action='store_true')
+        c.argument('paid_high_availability', action='store_true')
```

Comment:
The name is a little strange. Just my feeling. How about something like xxx tier, sku?

Reply:
Per PM, they wanna it to be uptime-sla, I will change it